src/components/structures/CompatibilityPage.js

```diff
@@ -48,19 +48,13 @@ module.exports = React.createClass({
                 </p>
                 <p>
                 { _t(
-                    'Please install <chromeLink>Chrome</chromeLink> or <firefoxLink>Firefox</firefoxLink> ' +
-                    'for the best experience.',
+                    'Please install <chromeLink>Chrome</chromeLink>, <firefoxLink>Firefox</firefoxLink>, ' +
+                    'or <safariLink>Safari</safariLink> for the best experience.',
                     {},
                     {
                         'chromeLink': (sub) => <a href="https://www.google.com/chrome">{sub}</a>,
                         'firefoxLink': (sub) => <a href="https://getfirefox.com">{sub}</a>,
```

Comment:
I think `https://firefox.com` is a slightly more official version, though they seem to redirect to the same place...